Saml.cs

```diff
@@ -208,24 +238,94 @@ public virtual string GetCompany()
 			return GetCustomAttribute("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/companyname")
 				?? GetCustomAttribute("User.CompanyName");
 		}
+
+	}
 
-		public string GetCustomAttribute(string attr)
-		{
-			XmlNode node = _xmlDoc.SelectSingleNode("/samlp:Response/saml:Assertion[1]/saml:AttributeStatement/saml:Attribute[@Name='" + attr + "']/saml:AttributeValue", _xmlNameSpaceManager);
-			return node == null ? null : node.InnerText;
-		}
+	public class SignoutResponse : BaseResponse
+	{
+		public SignoutResponse(string certificateStr, string responseString) : base(certificateStr, responseString) { }
+
+		public SignoutResponse(byte[] certificateBytes, string responseString) : base(certificateBytes, responseString) { }
+
+		public SignoutResponse(string certificateStr) : base(certificateStr) { }
+
+		public SignoutResponse(byte[] certificateBytes) : base(certificateBytes) { }
+
+		public string GetLogoutStatus()
+		{
+			XmlNode node = _xmlDoc.SelectSingleNode("/samlp:LogoutResponse/samlp:Status/samlp:StatusCode");

```

Comment:
I have to add the `_xmlNameSpaceManager` as following, otherwise it will throw error "_Namespace Manager or XsltContext needed. This query has a prefix, variable, or user-defined function._"
`XmlNode node = _xmlDoc.SelectSingleNode("/samlp:LogoutResponse/samlp:Status/samlp:StatusCode", _xmlNameSpaceManager);`